.circleci/scripts/config-template.yml

```diff
@@ -286,7 +286,7 @@ jobs: # Each project will have individual jobs for each specific task it has to
       - run:
           name: Install dotnet
           command: |
-            curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel Current
+            curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel lts
```

Comment:
`Current` is no longer a valid `channel` value. (and appears to be ignored)
The new name is `sts`, however, with the release of net8, I figured we'd want to use `lts`? @AlanRynne 

https://github.com/dotnet/install-scripts/blob/d7ff04d436ed8d76d6e95a8b37373f66d49648a2/src/dotnet-install.sh#L476

Reply:
Oh nvm... looks like `lts` is still referencing NET6 🤦‍♂️ not NET8
Going to change to `sts`